code/API_definitions/network_access_management.yaml

```diff
@@ -132,7 +132,7 @@ info:
     may be used.  If an end-user replaces the equipment, all existing isolated networks SHOULD be synced to the new
     device representing this mesh.  Alternatively, a synthetic hardware address may also be used if the specification
     for the address type allows.
-  version: wip
+  version: 0.1.0

```

Comment:
I'd stick with wip for now, until all of the commonalities and ICM stuff is finalized and we've incorporated it.